### PR TITLE
[Julia] Add links from exercise dir to concepts dirs

### PR DESCRIPTION
**Pull request by SaschaMann**
I find it easier to work on an exercise this way because I don't have to search through the ever-growing `concepts/` folder.

---

@ErikSchierboom this may be useful for other tracks too?

---

(depends on #2764 but it's ready for review. Only the last two commits 21dd2e6 and 3d4382c are relevant to this PR)

### CODE REVIEW

File: languages/julia/concepts/iterator-protocol/links.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "url": "https://invenia.github.io/blog/2020/11/06/interfacetesting/",
+        "description": "Development with Interface Packages",
+        "_why": "Julia's notion of informal interfaces is different to interfaces in many other languages. More experienced students may wonder how one can reliably define and test such interfaces. This post outlines how Invenia, and several of their open-sourced packages, do it in a production environment."
```
Comment (SaschaMann) [inline languages/julia/concepts/iterator-protocol/links.json:5]: See https://github.com/exercism/v3/issues/2499#issuecomment-727559809

File: languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+### Summary
+
+This exercise showed how to:
```
Comment (ErikSchierboom) [inline languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md:3]: For future reference: the `about.md` documents should not refer to the exercise as they should be consider stand-alone documents.

**Review by ErikSchierboom — COMMENTED** [review]
It looks like this doesn't add symlinks but copies of the concept files, is that correct?

File: languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md (hunk 2 above)
Comment (SaschaMann) [reply languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md:3]: > It looks like this doesn't add symlinks but copies of the concept files, is that correct?

Locally it does show them as symlinks, I think it's just how GitHub renders them?

Could you try checking out the branch locally and seeing if they're actually links?

Comment (ErikSchierboom) [reply languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md:3]: Doesn't look like it:

```
erik@VELPIE:~/v3$ gh pr checkout 2790
Already on 'julia/concept-symlinks'
Your branch is up to date with 'origin/julia/concept-symlinks'.
Already up to date.
erik@VELPIE:~/v3$ cd languages/julia/exercises/concept/
boolean-logic/      encounters/         fibonacci-iterator/
emoji-times/        exercism-matrix/    lasagna/
erik@VELPIE:~/v3$ cd languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction$ ls -al
total 16
drwxr-xr-x 2 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 ..
-rw-r--r-- 1 erik erik 2180 Nov 17 08:59 about.md
-rw-r--r-- 1 erik erik  123 Nov 17 08:59 links.json
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction$ cd ..
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts$ ls -al
total 12
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 ..
drwxr-xr-x 2 erik erik 4096 Nov 17 08:59 functions-introduction
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts$ cd ..
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta$ ls -al
total 24
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 4 erik erik 4096 Nov  6 12:29 ..
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 concepts
-rw-r--r-- 1 erik erik  372 Nov  6 12:29 config.json
-rw-r--r-- 1 erik erik  776 Nov  6 12:29 design.md
-rw-r--r-- 1 erik erik  151 Nov  6 12:29 example.jl
```

Comment (SaschaMann) [reply languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md:3]: Weird, I'll try again.

**Comment by ErikSchierboom** [comment]
This Friday (Jan 29th 2021), we're going to explode this repo back into the individual track repositories and archive this repo.

As such, this PR needs to be merged by Friday 9am UTC, or it will need to be manually reopened against the track repo.

**Pull request closed without merging**